src/server/api/auxiaProxyRouter.ts

```diff
@@ -190,67 +255,12 @@ export const buildAuxiaProxyRouter = (config: AuxiaRouterConfig): Router => {
             'gateDismissCount',
             'countryCode',
             'mvtId',
+            'should_show_legacy_gate_tmp',
```

Comment:
is this field required?

Reply:
Yes. The client side always provides it. This was just missing from the time I introduced it.